working_process.md

```diff
@@ -0,0 +1,40 @@
+# Working Process
+
+These guidelines applied to **all** CocoaHeads Russia projects.
+
+## Xcode and Swift Style Guidelines
+
+**We follow [these guidelines](https://github.com/raywenderlich/swift-style-guide) in coding**
+
+* Don't use acronyms and abbreviations
+* Use indentation == 2 in for *Tab width* and *Indent width*
+
+**Xcode → Preferences → Text Editing → Indentation**
+
+![Indentations](/resources/images/indentation.png)
+
+## Coding
+
+### iOS/macOS
+
+**Naming**
```

Comment:
This and all below is not working process in general. However, it's important information. I think we can write out it.

Reply:
Agree. Moved it to separate file `code_style.md`. In working process now there are only working process :)